packages/wrangler/src/vectorize/insert.ts

```diff
@@ -16,11 +21,10 @@ export function options(yargs: CommonYargsArgv) {
 			description: "The name of the Vectorize index.",
 		})
 		.options({
-			vectors: {
-				type: "array",
+			file: {
+				describe: "A file containing line separated json (ndjson) vector objects",
```

Comment:
```suggestion
				describe: "A file containing line separated Newline Delimited JSON (NDJSON) vector objects",
```

Reply:
Do we want to specify a maximum size in the help text too?

e.g.

`A file containing line separated Newline Delimited JSON (NDJSON) vector objects (max lines: ${VECTORIZE_UPSERT_BATCH_SIZE})`

Reply:
I'd be cool with this - it probably does make sense to make it our batch size (since that defeats the purpose of batching) but feel free to suggest something reasonable... maybe like 500k initially?